scripts/debug-publish-to-unity-package.bat

```diff
@@ -1,30 +1,42 @@
-@ECHO OFF
+
+
+SET SCRIPT_DIR=%~dp0
 
 echo Building project...
-pushd src/ChainSafe.Gaming.Unity
+pushd "%SCRIPT_DIR%\..\src\ChainSafe.Gaming.Unity"
 
 del obj /F /Q
 del bin /F /Q
+dotnet restore
 dotnet publish -c debug -f netstandard2.1 /property:Unity=true
 if %errorlevel% neq 0 exit /b %errorlevel%
 
 echo Restoring non-Unity packages...
 
-pushd ..\..
-dotnet restore
-popd
-
 echo Moving files to Unity package...
 
 pushd bin\debug\netstandard2.1\publish
 del Newtonsoft.Json.dll
 del UnityEngine.dll
+
+if exist "..\..\..\..\..\..\Packages\io.chainsafe.web3-unity.lootboxes" (
+    echo Directory exists, performing actions...
+    rmdir /s /q "..\..\..\..\..\..\Packages\io.chainsafe.web3-unity.lootboxes\Chainlink\Runtime\Libraries"
+    mkdir "..\..\..\..\..\..\Packages\io.chainsafe.web3-unity.lootboxes\Chainlink\Runtime\Libraries"
+    copy Chainsafe.Gaming.Chainlink.dll "..\..\..\..\..\..\Packages\io.chainsafe.web3-unity.lootboxes\Chainlink\Runtime\Libraries"
+    copy Chainsafe.Gaming.LootBoxes.Chainlink.dll "..\..\..\..\..\..\Packages\io.chainsafe.web3-unity.lootboxes\Chainlink\Runtime\Libraries"
+) else (
+    echo Directory does not exist, skipping actions.
+)
+
+del Chainsafe.Gaming.Chainlink.dll
+del Chainsafe.Gaming.LootBoxes.Chainlink.dll
+
 del Microsoft.CSharp.dll
 if not exist ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries mkdir ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\
 del ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\* /F /Q
 copy *.dll ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
-copy *.pdb ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
```

Comment:
Why we don't need pdb? It keeps all the debug info

Reply:
@kalambet I just updated with the code from the main publish, will add back